DQM/DTMonitorClient/src/DTBlockedROChannelsTest.cc

```diff
@@ -59,8 +59,9 @@ void DTBlockedROChannelsTest::fillChamberMap( DQMStore::IGetter & igetter, const
   // get the RO mapping
   context.get<DTReadOutMappingRcd>().get(mapping);
 
-  // fill the map of the robs per chamber
-  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=FEDNumbering::MAXDTFEDID; ++dduId) { //loop over DDUs
+  // fill the map of the robs per chamber 
+  // //FIXME: monitoring only real used FEDs
+  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=774; ++dduId) { //loop over DDUs
```

Comment:
Thank you. No problem for the slow answer, I understand the hardness and overload of your task.
This is not a permanent solution for DT, the non used FED IDs are booked for DT but have never been used, DT community is aware of this solution and requested this change: we just disable the monitoring of current non used FEDs.